docs/semgrep-cloud-platform/user-management.md

```diff
@@ -157,5 +157,24 @@ To **change a user's role**:
 You cannot change your own role.
 :::
 
+## Appendix: Token scopes
+
+When creating a new token, you are able to set a token's **scope**. Token scopes enable you to limit or grant permissions as necessary. Tokens are also autogenerated with appropriate scopes by Semgrep Cloud Platform based on the user's role or when onboarding a repository.
+
+The following table displays token scopes and their permissions:
+
+| Token scope | Send findings from a remote repository | Send findings from a local repository |  Send PR or MR comments | Connect to Semgrep API |
+| ----------- | ------- | --- | --| -- |
+| Agent (CI)  |   ✔️  Yes    |  ✔️ Yes  | ✔️  Yes [SME to confirm] | ✔️  Yes  |
+| Web API     |  ❌ No    |  ❌ No | ✔️  Yes [SME to confirm] | ✔️ Yes  |
+| Member      |    ❌ No   |  ✔️ Yes  | ❌ No |❌ No |
+
+The following table displays typical uses for token scopes:
+
+| Token scope |Typical uses |
+| ----------- | ----------- |
+| Agent (CI)  | Autogenerated by Semgrep Cloud Platform when onboarding (adding) a repository to Semgrep Cloud Platform. |
+| Web API     | Manually selected to access Semgrep's API or when additional permissions are required to enable PR or MR comments. | 
+| Member      | Autogenerated by Semgrep CLI when a member is logging in through Semgrep CLI. |
```

Comment:
(Sara to check if admin users are able to edit token scopes.)

Reply:
You can make edit Agent (CI) tokens and Web API tokens in the app. You cannot edit member tokens in the app. Might help to clarify in these docs that a member is an RBAC member so this only really applies to organizations that have RBAC.

Reply:
Added that bit in. Thank you!